metacatalog/models/eddy_data.py

```diff
@@ -9,5 +9,67 @@ class EddyData(Base):
 
     # columns
     entry_id  = Column(Integer, ForeignKey('entries.id'), primary_key=True)
-    
-    # TODO: put all the other columns here
\ No newline at end of file
+
+    # TODO: Datentypen überprüfuen, welche Werte sind nicht nullable? Gibt es weitere keys?
```

Comment:
Let's stick to English. Maybe @sihassl can say more. 
I would suggest: remove to do and open issue to not forget about this. Then we can merge without additional `nullable` set. Maybe we need to further discuss this (in an issue)